components/logins/android/src/main/java/mozilla/appservices/logins/DatabaseLoginsStorage.kt

```diff
@@ -248,13 +251,14 @@ class DatabaseLoginsStorage(private val dbPath: String) : AutoCloseable, LoginsS
     @Throws(LoginsStorageException::class)
     override fun importLogins(logins: Array<ServerPassword>): JSONObject {
         return writeQueryCounters.measure {
-            val s = JSONArray().apply {
-                logins.forEach {
-                    put(it.toJSON())
-                }
-            }.toString()
+            val builder = MsgTypes.PasswordInfos.newBuilder()
+            logins.forEach {
```

Comment:
Nit(can ignore if you want): How about we try do what [we're doing in Swift](https://github.com/mozilla/application-services/blob/ec0d019d1004269ad7921b8230e50741541eaf63/components/fxa-client/ios/FxAClient/FxAccountDevice.swift#L82-L88) instead here and [there](https://github.com/mozilla/application-services/blob/ec0d019d1004269ad7921b8230e50741541eaf63/components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt#L554) (I guess `capabilitiesToBuffer` can go). It seems like Kotlin also has the capability to [add extensions to typed arrays](https://kotlinlang.org/docs/reference/extensions.html#extension-functions). This would set a good clean example for the future.

Reply:
Done. Hope I get it right :)